cpp/include/cugraph_c/cugraph_api.h

```diff
@@ -42,16 +43,16 @@ extern int data_type_sz[];
 
 /* C stub declarations */
 
-typedef struct cugraph_raft_handle_ {
-  int allign_;
-} cugraph_raft_handle_t;
+typedef struct cugraph_handle_ {
```

Comment:
I like this change. I noticed that this type is described in the API doxygen comments as a "Handle for accessing resources".  Would a more self-documenting name be `cugraph_resource_handle_t`?  This would also make it less likely to be confused with a handle to a `cugraph_graph_t` or something similar.